Feature/react recap project (#340)
neuefische/web-exercises: sessions/recap-project-react/part-2/hints.md

# Part 2 - Hints

<details>
<summary>💡 Hint: State </summary>

> Placing a state only where its needed can reduce complexity a lot.

Reviewer: ```suggestion
If you only place a state where it is needed, you can significantly reduce complexity.
```
